js/index.js

```diff
@@ -1,6 +1,30 @@
-// Iteration 1: Names and Input
+let hacker1 = "Carol";
+console.log(`The driver's name is ${hacker1}`);
+let hacker2 = "Junior";
+console.log(`The navigator's name is ${hacker2}`);
 
-// Iteration 2: Conditionals
+//Iteration 2
+if(hacker1.length > hacker2.length){
+  console.log(`The driver has the longest name, it has ${hacker1.length} characters.`);
+} else if (hacker1.length < hacker2.length){
+  console.log(`It seems that the navigator has the longest name, it has ${hacker2.length} characters.`);
+} else if (hacker1.length === hacker2.length){
+  console.log(`Wow, you both have equally long names, ${hacker1.length} characters`);
+} else {
+  console.log("Name Error");
+}
```

Comment:
cara muito bom vc ter colocado essa última condição `Name Error`. Acho que foi o único que eu vi se preocupando com esse "corner case". Porém, eu ACHO que nunca vai chegar nesse ponto, porque se o tamanho não for maior, nem menor, nem igual, eu acho que daria erro antes ok? Qualquer coisa falamos sobre isso depois, mas a ideia foi muito boa!